spec/controllers/address_controller_spec.rb

```diff
@@ -2,28 +2,23 @@
 
 require "rails_helper"
 
-RSpec.describe AddressController, :member, type: :controller do
-  let!(:current_app) do
-    SnapApplication.create!(
-      user: member,
-      street_address: "123 Fake St",
-      city: "Springfield",
-      county: "Genesee",
-      zip: "12345",
-      state: "MI",
-    )
-  end
-
+RSpec.describe AddressController, type: :controller do
   let(:step) { assigns(:step) }
 
   describe "#edit" do
     it "assigns the correct step" do
+      session[:snap_application_id] = current_app.id
```

Comment:
In general I want to keep this app as similar to GCF as possible, but the use of `let` and `before` in the GCF codebase is probably something I would not choose to duplicate.

All that being said, I agree that this is a place where it makes sense. Done at 8b9527e 😉